__tests__/app.test.js

```diff
@@ -125,6 +125,90 @@ describe("/articles/:article_id/comments", () => {
         });
       });
   });
+  test("POST:201 inserts a new comment of a given article_id to the db and sends the new comment back to the client for a username that is in the users table", () => {
+    const newComment = {
+      author: "butter_bridge",
+      body: "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+    };
+    return req(app)
+      .post("/api/articles/6/comments")
+      .send(newComment)
+      .expect(201)
+      .then((res) => {
+        expect(res.body.comment).toHaveProperty("body");
+        expect(res.body.comment).toHaveProperty("votes");
+        expect(res.body.comment).toHaveProperty("author");
+        expect(res.body.comment).toHaveProperty("created_at");
+        expect(res.body.comment).toHaveProperty("comment_id");
```

Comment:
This would be a stronger test if you used toHaveProperty to also check the data type e.g. `expect(res.body.comment).toHaveProperty("body", expect.any("String");`
        
 toHaveProperty can take another property representing what you want the value to be, and this can be used to check the data type too!